src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAccountManager.java

```diff
@@ -1491,6 +1492,20 @@ public UserSummaryDTO convertToUserSummaryObject(final RegisteredUserDTO userToC
     return this.dtoMapper.map(userToConvert, UserSummaryDTO.class);
   }
 
+  /**
+   * Helper method to convert a user object into a more detailed summary object depending on the dto provided.
+   *
+   * @param userToConvert    - full user object.
+   * @param detailedDTOClass - The level of detail required for the conversion
+   * @return a summarised object with reduced personal information
+   */
+  public UserSummaryWithEmailAddressAndGenderDTO convertToUserSummaryWithEmailAddressAndGenderObject(
```

Comment:
If you make the new DTO extend `UserSummaryWithEmailAddressDTO`, you can get rid of this function and simply use the one below (or a modified version of it that uses generics, which would be better).